docs/_articles/en/faq.md

```diff
@@ -56,8 +56,7 @@ A: You must:
 #### Q: How can I use `Salesforce CLI Scanner` in my CI/CD?
 A: You can use the `sfdx scanner:run` command in any scripts used by your CI/CD. You'll also probably want to do the following:
 - Use the `-o/--outfile` flag to write your results to a file, so you'll have an artifact of the results.
-- Use the -s/--severity-threshold flag to cause a non-zero exit code if any violations meet or exceed the provided value, since many CI/CD frameworks care about such things.
-- `-v/--violations-cause-error` flag is deprecated please use `-s/--severity-threshold`. 
+- Use the `-s/--severity-threshold` flag to cause a non-zero exit code if any violation's normalized severity meets or exceeds the provided value, since many CI/CD frameworks care about such things.
```

Comment:
Documentation needs to go to devdoc branch.. @rmohan20 can you please work with Josh to establish a new process for this?